### DISCUSSION TITLE
Neopixel module

### DISCUSSION THREAD

**Discussion by cve2022** [micropython/micropython#10126] [Libraries & Drivers] [answered]
Sorry if this is a repeated question.

Where do I find the neopixel.py module? The [link in the documentation](https://github.com/micropython/micropython/blob/master/drivers/neopixel/neopixel.py) seems to be broken (I get a 404 error).

Thanks.

**Comment by robert-hh** [comment, marked as answer]
https://github.com/micropython/micropython-lib/tree/038b4ac6572d0d5b4c18148dc7d7fdc026369fa4/micropython/drivers/led/neopixel

**Reply by cve2022** [reply]
Robert, thank  you very much.

**Comment by jimmo** [comment]
Thanks for reporting -- raised https://github.com/micropython/micropython/pull/10161 to fix.
